addons/web/static/src/core/utils/draggable_hook_builder.js

```diff
@@ -63,7 +63,7 @@ import { debounce, setRecurringAnimationFrame } from "@web/core/utils/timing";
  * @property {Function} helpers.execHandler
  */
 
-import { useEffect, useEnv, useExternalListener, onWillUnmount, reactive } from "@odoo/owl";
+const { useEffect, useEnv, useExternalListener, onWillUnmount, reactive } = owl;
```

Comment:
Please don't do this. The `@odoo/owl` syntax is used by the typescript integration to provide type hints.